src/frontends/paddle/src/op/less_than.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <map>
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs less_than(const NodeContext& node) {
+    auto x = node.get_input("X");
+    auto y = node.get_input("Y");
+
+    auto axis = node.get_attribute<int>("axis");
+
+    PADDLE_OP_CHECK(node, x.get_partial_shape().rank().is_static(), "elementwise_ops: X rank must be static!");
+    PADDLE_OP_CHECK(node, y.get_partial_shape().rank().is_static(), "elementwise_ops: Y rank must be static!");
```

Comment:
```suggestion
    PADDLE_OP_CHECK(node, y.get_partial_shape().rank().is_static(), "less_then: Y rank must be static!");
```

Reply:
used a common function for all elementwise related Op conversions.